WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -689,39 +692,42 @@ class MySiteFragment : Fragment(),
                 isDomainCreditAvailable = false
             }
             RequestCodes.PHOTO_PICKER -> if (resultCode == Activity.RESULT_OK && data != null) {
-                if (data.hasExtra(PhotoPickerActivity.EXTRA_MEDIA_ID)) {
-                    val mediaId = data.getLongExtra(PhotoPickerActivity.EXTRA_MEDIA_ID, 0).toInt()
-                    showSiteIconProgressBar(true)
-                    updateSiteIconMediaId(mediaId)
-                } else {
-                    val mediaUriStringsArray = data.getStringArrayExtra(
-                            PhotoPickerActivity.EXTRA_MEDIA_URIS
-                    )
-                    if (mediaUriStringsArray.isNullOrEmpty()) {
-                        AppLog.e(
-                                UTILS,
-                                "Can't resolve picked or captured image"
+                if (!handleMediaPickerResultForStories(data)) {
```

Comment:
this whole block here was just put into a conditional: `handleMediaPickerResultForStories` will return `true` if it ended up recognizing the request and handling it as part of a MediaPicker request for Stories. If not, then business as usual (i.e. the code as it was) is executed.